### DISCUSSION TITLE
Views [Apr 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5573] [Discord Threads]
> 💁 **o1lab**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Ninoprs**  
#### could I also make nocodb just the form endpoint, so the html the form has a nocodb link as a submit value?  
#  
> 🙋‍♂️ **Ninoprs**  
#### that way one could make more complext descriptions etc. since the default view is somewhat limited in it's customizability  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1089913041371537479)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
